js/src/gitutil.ts

```diff
@@ -18,10 +18,16 @@ export interface RepoStatus {
 }
 
 export async function currentRepo() {
-  const git = simpleGit();
-  if (await git.checkIsRepo()) {
-    return git;
-  } else {
+  try {
+    const git = simpleGit();
+    if (await git.checkIsRepo()) {
+      return git;
+    } else {
+      return null;
+    }
+  }
+  catch  (e) {
+    console.warn('Error occurred while checking if current directory is a git repo.', e);
```

Comment:
@ankrgyl - Not sure if this will be too noisy.  Can also drop the logging if it's not helpful.

Reply:
I think it's ok to drop, to be consistent with the Python implementation. We can always add it in to debug further if needed.

Out of curiosity, what did it print in your use case?

Reply:
This is the error with stack trace:
```
Error: Error: spawn git ENOENT
    at Process.ChildProcess._handle.onexit (node:internal/child_process:286:19)
    at onErrorNT (node:internal/child_process:484:16)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at Object.action (/<internal path>/node_modules/.pnpm/braintrust@0.0.85/node_modules/braintrust/dist/index.js:7460:25)
    at PluginStore.exec (/<internal path>/node_modules/.pnpm/braintrust@0.0.85/node_modules/braintrust/dist/index.js:7485:25)
    at /<internal path>/node_modules/.pnpm/braintrust@0.0.85/node_modules/braintrust/dist/index.js:4956:51
    at new Promise (<anonymous>)
    at GitExecutorChain.handleTaskData (/<internal path>/node_modules/.pnpm/braintrust@0.0.85/node_modules/braintrust/dist/index.js:4954:16)
    at GitExecutorChain.<anonymous> (/<internal path>/node_modules/.pnpm/braintrust@0.0.85/node_modules/braintrust/dist/index.js:4938:44)
    at Generator.next (<anonymous>)
    at fulfilled (/<internal path>/node_modules/.pnpm/braintrust@0.0.85/node_modules/braintrust/dist/index.js:3761:24)
```

Reply:
Removed warning: https://github.com/braintrustdata/braintrust-sdk/pull/83/commits/1511047026dfe0ecc39ea727f5d0342d73a6784e